h3/src/client/builder.rs

```diff
@@ -104,6 +104,12 @@ impl Builder {
         self
     }
 
+    /// Enables the extended CONNECT protocol required for various HTTP/3 extensions.
+    pub fn enable_connect(&mut self, value: bool) -> &mut Self {
```

Comment:
Shouldn't the method be called `enable_extended_connect`?

Reply:
There's no API stability at the moment while at 0.0.x. All upgrades are manual.

Reply:
Renamed in `client::builder()` and `server::builder()`.